apps/aecore/test/aec_chain_sim.erl

```diff
@@ -395,8 +382,24 @@ remove_meck() ->
 %$% gen_server implementation
 %%%===================================================================
 
+setup_log_file() ->
```

Comment:
I appreciate your effort to solve logging, but at the same time I have to ask a few questions:

1) What the purpose? In the common_test mode, I can see a well-structured HTML report (don't need to reinvent);
2) To log all simulator flow? It can be large (if you could check that each `?LOG("chain_req(~p)", [Req]),` produces a lot of data as IO);
3) I can be wrong, but when we need a report - we use the SUITE tooling, to see the actual flow in the report. I propose to remove that as IO is large and we also have a lot of cases outside this contract scope (at least we are going to generate intensive parent chain behaviors for our parent DB capacity measurements).

P.S. The problem before was as a flag "user" made our parent chain generation extremely slow and broken even for 100 blocks generation case, so I propose to take the responsibility of logging as a privilege of the simulator managing process and make it more clean here.

Reply:
Actually, after a 1-month master will be renamed as "main" within all GitHub , are you ready my friends to fix broken scripts? )))

Reply:
@erlmachinedev the HC staking contract runs under eunit where no logging sink is set.
@radrow only create this logger in eunit

Reply:
I've seen it (I reviewed) because eunit itself doesn't suppose to write large flow execution reports into IO. Don't need your own logger - need log your particular unit test, log in your own scope. This makes the simulator slower a lot

Reply:
@gorbak25 moreover, the default io is redirected into file, which removes log from HTML suite report at all

Reply:
fix'd